### PR TITLE
installation via RVM

### PR DESCRIPTION
**Pull request by tuxmea**

### CODE REVIEW

File: spec/classes/hdm_rvm_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  on_supported_os.each do |os, os_facts|
+    context "on #{os} using rvm" do
+      let(:facts) { os_facts }
+      let(:params) do
+        {
+          'method' => 'rvm',
+        }
+      end
+
+      it { is_expected.to compile }
```
Comment (bastelfreak) [inline spec/classes/hdm_rvm_spec.rb:15]:
```suggestion
      it { is_expected.to compile.with_all_deps }
```

File: manifests/rvm.pp [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        package { 'centos-release-scl':
+          ensure => present,
+        }
+
+        package { 'devtoolset-7':
+          ensure => present,
+        }
+        $exec_prefix = 'scl enable devtoolset-7 '
+        exec { 'update sqlite':
+          command => 'yum install -y https://kojipkgs.fedoraproject.org//packages/sqlite/3.8.11/1.fc21/x86_64/sqlite-devel-3.8.11-1.fc21.x86_64.rpm https://kojipkgs.fedoraproject.org//packages/sqlite/3.8.11/1.fc21/x86_64/sqlite-3.8.11-1.fc21.x86_64.rpm',
```
Comment (bastelfreak) [inline manifests/rvm.pp:52]: I think we can use the package resource and provide the link to the package?

Comment (tuxmea) [reply manifests/rvm.pp:52]: No. we need to install both packages at once as we are upgrading from sqlite 3.7 to 3.8.

**Review by rwaffen — COMMENTED** [review]
LGTM

**Pull request merged**